WooCommerce/Classes/System/WooNavigationController.swift

```diff
@@ -75,4 +111,15 @@ private extension WooNavigationControllerDelegate {
     func configureBackButton(for viewController: UIViewController) {
         viewController.removeNavigationBackBarButtonText()
     }
+
+    /// Set up toolbar for the view controller to display the offline message,
+    /// and listen to connectivity status changes to change the toolbar's visibility.
+    ///
+    func configureOfflineBanner(for viewController: UIViewController, in navigationController: WooNavigationController) {
+        viewController.toolbarItems = toolbarItemsForOfflineBanner
+        navigationController.toolbar.barTintColor = .gray
+
+        let connected = ServiceLocator.connectivityObserver.isConnectivityAvailable
```

Comment:
We should probably inject `Service.Locator.connectivityObserver` with a default parameter!